episodes/files/my_first_bash_script.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
+# This script loops through .txt files, returns the
```

Comment:
Small issue with formatting of this comment - doesn't match what is in the lesson